src/breadcrumbs/breadcrumbs.scss

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../icon/index';
+
+@mixin iui-breadcrumbs {
+  @include iui-reset;
+  @include iui-font-family;
+  display: flex;
+  align-items: center;
+  
+  .iui-breadcrumbs-list {
+    @include iui-reset;
+    list-style-type: none;
+    display: flex;
+    align-items: center;
+    user-select: none;
+
+    .iui-breadcrumbs-item a,
+    .iui-breadcrumbs-item .iui-button .iui-label {
+      overflow: hidden;
+      white-space: nowrap;
+      text-overflow: ellipsis;
+      max-width: $iui-3xl * 2;
+      text-align: left;
+    }
+
+    .iui-breadcrumbs-item {
+      .iui-icon {
+        vertical-align: middle;
+        @include iui-icons-small;
+      }
+
+      a {
+        @include iui-anchor;
+      }
+
+      a,
+      .iui-ellipsis {
+        margin: 0 $iui-sm;
+      }
+
+      button {
```

Comment:
Not sure these selectors are good idea.. Maybe not valid, but if breadcrumb has some custom button inside, we would override rules..

Reply:
No longer targeting elements, replaced with `.iui-button` and `.iui-anchor`.  Also added `>` to be more specific.